examples/pie_and_polar_charts/polar_error_caps.py

```diff
@@ -0,0 +1,26 @@
+"""
+=================================
+Error bar rendering on polar axis
+=================================
+
+Demo of error bar plot in polar coordinates.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+fig = plt.figure(figsize=(10, 10))
+ax = plt.subplot(111, projection='polar')
```

Comment:
This would be more idiomatic, but not a big deal:
```suggestion
ax = fig.add_subplot(projection='polar')
```

Reply:
I will fix it, I am taking some time to think of a more elaborate gallery example :-)

Reply:
Ok, I could not think of something better. I have just added overlapping examples, including your suggestions.